.github/workflows/push-to-develop.yml

```
@@ -0,0 +1,39 @@
+name: push-to-develop
+
+on:
+  push:
+    branches:
+      - master
```

```suggestion
      - develop
```
Should it be develop?

---

Ah! Nice catch!